openmetadata-docs/content/deployment/upgrade/docker.md

```diff
@@ -22,39 +22,123 @@ It is adviced to go through [openmetadata release notes](/deployment/upgrade#bre
-```
-wget https://github.com/open-metadata/OpenMetadata/releases/download/{version}-release/docker-compose.yml
-```
-or if you wish to use postgres as the database
-```
-wget https://github.com/open-metadata/OpenMetadata/releases/download/{version}-release/docker-compose-postgres.yml
-```
+Database volumes were just introduced in the 0.13.0 feature release. This means that when moving from 0.12.3 to 0.13.1, we'll
+need to do some extra steps to ensure that all your metadata is safe and sound. In this section, we'll guide you step by step on
+what you'll need to do to:
 
-### 2. Backup your Data [IMPORTANT]
+1. Backup your data,
+2. Add volumes and port mapping to the databases and restore your data,
+3. Start OpenMetadata with the 0.13.1 release.
 
-Make a backup of your database. You can find the steps to follow [here](/deployment/upgrade//deployment/backup-restore-metadata#backup-metadata). Please note this is an important step as it would allow you to revert to a stable state if any issues were to happen during your upgrade.
+These steps are based on MySQL, but are analogous for Postgres.
 
-### 3. Add Volumes and Publish Ports
+Please, validate the process in your development or staging environment first before going into PROD.
 
-Update the docker compose file you just downloaded to add persistent volume for MySQL. You can follow the steps [here](/deployment/docker/volumes#docker-volumes). If you had previously configured volumes for MySQL and the ingestion container make sure the 0.13.0 compose file is referencing the correct volume so that 0.12.x data will be available when upgrading to 0.13.0.
 
-**Note:** in 0.13.0 we are not publishing MySQL ports to the host container. If you wish to do so you will need to update the MySQL service in the compose file to have the following configuration
-```yaml
-services:
-  mysql:
-    ...
-    ports:
-      - "3306:3306"
-    ...
-```
+<Note>
+
+This guide is specific to installations using the default docker compose file we shipped on the 0.12.3 release. The
+whole process is about making sure that all the data will be properly backed up considering that there is no
+port mapping or volumes available.
+
+If your docker compose already had volumes for the database, then upgrading the version of OpenMetadata and the
+Ingestion container should be good to go (as shown in step 4 below).
 
-### 5. Stop, Remove and Start your Containers
-Stop and remove (without deleting volumes) the already running containers. Then run the `docker compose up -d` command on the new compose file.
+</Note>
+
+### 1. Backup 0.12.3 data
+
+1. Get inside the `openmetadata_ingestion` container with `docker exec -it openmetadata_ingestion bash`.
+2. Create a virtual environment to install an upgraded `metadata` version to run the backup command:
+   1. `python -m venv venv`
+   2. `source venv/bin/activate`
+   3. `PIP_USER=false pip install openmetadata-ingestion~=0.13.1`
```

Comment:
Should the version here be `0.13.0.3` or `0.13.1.0` ?

Reply:
I believe it should be good. the `~=` will pick up the latest patched version

Reply:
so it will start by 0.13.1.0, but will increase to 0.13.1.1, 0.13.1.2 as long as we keep publishing fixes